DotNET/Complex Flow Examples/merge-different-file-types.cs

```diff
@@ -0,0 +1,96 @@
+using Newtonsoft.Json.Linq;
+using System.Text;
+
+/* In this sample we will show how to merge different file types together as
+* discussed in https://pdfrest.com/solutions/merge-multiple-types-of-files-together/.
+* Specifically we will be uploadng an image file to the /pdf route and capturing
+* the output ID, uploading a powerpoint file to the /pdf route and capturing the
+* output ID and then passing both of those IDs to the /merged-pdf route to get
+* a singular output PDF combining the two inputs
+
+* Note that there is nothing special about an image and a powepoint file and
+* this sample could be easily used to convert and combine any two file types
+* that the /pdf route takes as inputs
+*/
```

Comment:
I propose we break up the final sentence of the first chunk to improve readability:

```suggestion
/* In this sample, we will show how to merge different file types together as
* discussed in https://pdfrest.com/solutions/merge-multiple-types-of-files-together/.
* First, we will upload an image file to the /pdf route and capture the output ID.
* Next, we will upload a PowerPoint file to the /pdf route and capture its output
* ID. Finally, we will pass both IDs to the /merged-pdf route to combine both inputs
* into a single PDF.

* Note that there is nothing special about an image and a PowerPoint file, and
* this sample could be easily used to convert and combine any two file types
* that the /pdf route takes as inputs.
*/
```
Also added a bit more punctuation.